3DRotation/3DRotationJSONSchema.json

```diff
@@ -254,11 +254,12 @@
       "description" : "An identifier that can be used either by the developer or researcher for custom mapping.",
       "type" : "string"
     },
+    "schemaIdentifierPath" : {
+      "description" : "URL linking to the version of the schema against which these data should validate.",
+      "type" : "string"
+    },
```

Comment:
@akaat This won't work. You are just introducing a *different* key that is not in the encoding.